habitat-baselines/habitat_baselines/rl/multi_agent/pop_play_wrappers.py

```diff
@@ -204,10 +294,23 @@ def _merge_step_outputs(self, get_step):
                     continue
                 agent_step_data[k].append(v)
         obs = TensorDict(obs)
+        new_agent_step_data = {}
         for k in agent_step_data:
-            agent_step_data[k] = torch.cat(agent_step_data[k], dim=-1)
-        agent_step_data = dict(agent_step_data)
+            new_name = "index_len_" + k
```

Comment:
I don't understand what is happening here. Could you write some comments?

Reply:
Updated in latest commit